### ISSUE TITLE
What if I can't answer?

### ISSUE THREAD

**Issue by andrewm4894** [LAION-AI/Open-Assistant#814]
This is in German and I don't know German so can't really say if it's spam or not. Should I be able to somehow say "I don't know"

Maybe I could skip it but I wonder if collecting explicit I don't know could also be useful in some way.

![Screenshot_20230117-224552](https://user-images.githubusercontent.com/2178292/213028947-f2edb720-ba7b-4e07-a614-eb84d7899f74.png)

**Comment by othrayte** [comment]
We discussed this a little bit in the design of adding a reason to the skip workflow (#310). We ended up deciding to go with free text for now rather than trying to have a predefined list of reasons that we could act on. As I understand it this is just an initial workflow for the MVP whilst we learn more about the issues we'll face.

In the case of handling the language we discussed in a previous web video meeting that we would in future like to add some knowledge of the language the user understands and try to not even present those uses with tasks they can't read. Once we move to doing that we will probably need some language flagging system to handle the odd case that they do get one that they can't read.
